db/test/test_lastfm_token.py

```diff
@@ -0,0 +1,95 @@
+from __future__ import absolute_import, print_function, unicode_literals
+from db.testing import DatabaseTestCase
+import logging
+from datetime import timedelta
+import db
+from db.lastfm_user import User
+from db.lastfm_token import Token, TOKEN_EXPIRATION_TIME
+from sqlalchemy import text
+import uuid
+
+
+class TestAPICompatTokenClass(DatabaseTestCase):
+
+    def setUp(self):
+        super(TestAPICompatTokenClass, self).setUp()
+        self.log = logging.getLogger(__name__)
+
+        # Create a user
+        uid = db.user.create("test")
+        self.assertIsNotNone(db.user.get(uid))
+        with db.engine.connect() as connection:
+            result = connection.execute(text('SELECT * FROM "user" WHERE id = :id'),
```

Comment:
You're mixing quote styles here. If you need quotes in a string (as in our `"user"` case), I'd prefer we use triple-quotes for strings.
I prefer to write queries in multiple parts for readability, e.g.
```python
query = text("""
    SELECT a
         , b
         , c 
      FROM "user" 
     WHERE id = :id""")
result = connection.execute(query, {"id": uid})
```

Note the alignment of the SQL queries - it's very specific. keywords are right-aligned, column names and expressions are left-aligned, and commas go on a new line (this is so that if you add another WHERE clause or column to select, you only add 1 new line, instead of changing another to add a comma at the end of the line)